osf/external/chronos.py

```diff
@@ -83,10 +83,17 @@ def serialize_manuscript(cls, journal_id, preprint, status=ChronosSubmissionStat
 
     @classmethod
     def serialize_user(cls, user):
+        if not bool(user.given_name) and not bool(user.family_name):
+            raise ValueError(
+                'Cannot submit because user {} requires a given and family name'.format(user.given_name if user.given_name and user.family_name else user.fullname)
+            )
+
         return {
             'CHRONOS_USER_ID': user.chronos_user_id,
             'EMAIL': user.username,
             'GIVEN_NAME': user.given_name if str(user.given_name) and str(user.family_name) else user.fullname,
```

Comment:
Now that we send them all the names, do we still need these conditionals? Can we do `user.given_name` only?